app/vmagent/remotewrite/relabel.go

```diff
@@ -37,6 +40,38 @@ func CheckRelabelConfigs() error {
 	return err
 }
 
+func initRelabelConfigs() {
+	rcs, err := loadRelabelConfigs()
+	if err != nil {
+		logger.Fatalf("cannot initialize relabel configs: %s", err)
+	}
+	allRelabelConfigs.Store(rcs)
+	if rcs.isSet() {
+		metrics.GetOrCreateCounter("vmagent_relabel_config_last_reload_successful").Set(1)
```

Comment:
I think, it's better to keep metrics init consistent.  Previously, it was clear how it was inited via direct `var.. ` declaration. Now it's hard to find.

 I have the following suggestions:
1) do not over-complicate exist solution and provide simple bugfix with setting `saCfgSuccess` to 1 as it already implemented for 
```
       relabelConfigSuccess.Set(1)
	relabelConfigTimestamp.Set(fasttime.UnixTimestamp())
```

2) introduce new `initMetrics` function, that must be called on non-zero configs and init all metrics at one place with `New...` methods.